src/fu-util-common.c

```diff
@@ -1713,7 +1713,7 @@ fu_util_plugin_to_string(FwupdPlugin *plugin, guint idt)
 	const gchar *hdr;
 	guint64 flags = fwupd_plugin_get_flags(plugin);
 
-	fwupd_codec_string_append(str, idt, fwupd_plugin_get_name(plugin), NULL);
+	fwupd_codec_string_append(str, idt, "Name", fwupd_plugin_get_name(plugin));
```

Comment:
Urgh, this is my fault -- but we were using the name as a "header" so the correct fix is

    -fwupd_plugin_get_name(plugin), NULL
    +fwupd_plugin_get_name(plugin), ""

Reply:
fixed